packages/contracts/contracts/core/permission/PermissionManager.sol

```diff
@@ -223,11 +223,7 @@ contract PermissionManager is Initializable {
     /// @param _where The address of the target contract for which `who` recieves permission.
     /// @param _who The address (EOA or contract) owning the permission.
     /// @param _permissionId The permission identifier.
-    function _grant(
-        address _where,
-        address _who,
-        bytes32 _permissionId
-    ) internal {
+    function _grant(address _where, address _who, bytes32 _permissionId) internal {
```

Comment:
We have a code formatter war here

Reply:
Mathias fixed it here: https://github.com/aragon/core/pull/201#pullrequestreview-1261366206
Please approve and then I will merge it in.